plugins/@grouparoo/spec-helper/src/lib/factories/source.ts

```diff
@@ -1,15 +1,21 @@
 import { loadPath } from "../loadPath";
 import faker from "faker";
 import AppFactory from "./app";
-import { Source } from "@grouparoo/core";
+import { Source, GrouparooModel } from "@grouparoo/core";
+import ModelFactory from "./model";
 
-const data = async (props = {}) => {
+const data = async (props: { modelId?: string } = {}) => {
   const { GrouparooModel } = await import(`@grouparoo/core/${loadPath}`);
+  const model =
+    (props.modelId
+      ? await GrouparooModel.findOne({ where: { id: props.modelId } })
+      : await GrouparooModel.findOne()) ??
```

Comment:
See my comment in `plugins/@grouparoo/spec-helper/src/lib/factories/destination.ts`